requirements.in

```diff
@@ -1,2 +1,2 @@
-salt-ssh==3002.9
+salt
```

Comment:
I think it would be good if we still locked, to make clear that upgrading is a major thing?

Reply:
Good idea.

Reply:
Done